tests/dummy/config/environment.js

```diff
@@ -48,8 +48,8 @@ module.exports = function(environment) {
 
   if (environment === 'production') {
     ENV.locationType = 'hash';
-    ENV.baseURL = '/ember-tooltips/';
-    ENV.baseURL = '/ember-tooltips';
+    ENV.rootURL = '/ember-tooltips/';
+    ENV.rootURL = '/ember-tooltips';
```

Comment:
good fixes. If this PR doesn't land because we decide to support 1.13 let's do this in a separate PR